### DISCUSSION TITLE
我的mac和windows电脑在公司中不同的ip地址，不同的Vlan下找不到对方

### DISCUSSION THREAD

**Discussion by 949331213** [localsend/localsend#1210] [L-简体中文]
我的两台电脑一台在10.16.19.**    
    另一台电脑设备在10.16.17.**
他们之间并不能互相发现，但是我选择输入地址是可以发现并传输的，所以想请开发者优化一下这个问题

**Comment by loctempt** [comment]
发现协议是基于multicast的，传输是直连，估计是vlan没配置multicast的转发，应该找网管改改路由配置

**Reply by virtual163** [reply]
你网应该是24位硽吗吧，所以他找不到不同网段的设备

**Reply by 949331213** [reply]
是的，子网掩码是255.255.254.0，导致识别为不同网段了，不过我们在使用IP方式是可以联通传输的
